Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -135,13 +146,15 @@ - (void)urlSessionTask:(NSURLSessionTask *)sessionTask setState:(NSURLSessionTas
 
     NSURL *url = [[sessionTask currentRequest] URL];
 
-    if (url == nil)
```

Comment:
Yikes, were these autoformatted or something you noticed? If the latter, should def look into adding a rule for this in swiftlint.

Reply:
Something I noticed. Not sure who has the habit of not using curly braces for one-line ifs, but yeah I really don't like it.

Can SwiftLint check ObjC code though?

Reply:
Ah yes, we'd need clang-format or clang-tidy for that.

Reply:
Guilty as charged ;)